README.md

```diff
@@ -37,31 +38,41 @@ NOTE: To interact with the OpenAI API, you need to either get and store an OPENA
     ```bash
     $ mvn -Pnative package
     ``` 
-    This produces a Linux self-contained executable version of the seachbot! 
+    This produces a Linux self-contained executable version of the searchbot! 
 
-4. Interact with the seachbot: 
+    <!-- Does it? Surely it depends on the platform? -->
+
+4. Interact with the searchbot: 
     ```bash
     $ ./target/searchbot “Why should I visit Lviv?”
     ```
 
-## Apply Profile-Guided Optimizations to Create an Optimized Native Executable
+## Apply Profile-Guided Optimization to Create an Optimized Native Executable
 
-[Profile-Guided Optimizations](https://www.graalvm.org/latest/reference-manual/native-image/optimizations-and-performance/PGO/) is the technique fetching the profiling information to the AOT compiler to improve the performance and file size of a native executable.
+[Profile-Guided Optimization](https://www.graalvm.org/latest/reference-manual/native-image/optimizations-and-performance/PGO/) is a technique that brings profile information to an AOT compiler to improve the quality of its output in terms of performance and size.
 
-1. Build an “instrumented” executable and run it to gather profiles:
+1. Build an “instrumented” executable:
     ```bash
-    $ native-image -jar ./target/searchbot-1.0-jar-with-dependencies.jar --pgo-instrument -o ./target/searchbot-instrumented
+    $ native-image 
+          -jar ./target/searchbot-1.0-jar-with-dependencies.jar \
+          --pgo-instrument \
+          -o ./target/searchbot-instrumented
     ```
+
+2. Run the executable to gather a profile:
     ```bash
     $ ./target/searchbot-instrumented “What is the weather today?”
     ```
  
-2. Build a PGO-optimized executable that will pick up the  gathered profiles, and  run: 
+3. Build a PGO-optimized executable using the profile, then run it: 
     ```bash
-    $ native-image -jar ./target/searchbot-1.0-jar-with-dependencies.jar --pgo -o ./target/searchbot-pgo-optimized
+    $ native-image \
```

Comment:
I missed adding the `--enable-sbom` option. Likely, I'll add more optimization flags (advice from Shaun Smith)

Reply:
That should be a separate commit.